src/5.0_to_5.1/__test_resources__/smallContentServer.ts

```diff
@@ -0,0 +1,1240 @@
+                        isDirectory: false,
+                        owners: ["admin"],
+                        readers: ["admin", "ROLE_USER"],
+                        timestamp: 1669901737180,
+                        lastEditor: "admin",
+                        canRead: true,
+                        canWrite: true,
+                      },
+                    },
+                    "14f": {
+                      entry: {
+                        isDirectory: true,
+                        owners: ["admin"],
+                        readers: ["admin"],
+                        timestamp: 1669285685624,
+                        lastEditor: "admin",
+                        canRead: true,
+                        canWrite: true,
+                      },
+                      children: {
+                        "14f_metadata": {
+                          entry: {
+                            content: '{"name":"Too many columns"}',
+                            isDirectory: false,
+                            owners: ["admin"],
+                            readers: ["admin"],
+                            timestamp: 1670334665557,
+                            lastEditor: "admin",
+                            canRead: true,
+                            canWrite: true,
+                          },
+                        },
+                      },
+                    },
+                    "31b": {
+                      entry: {
+                        isDirectory: true,
+                        owners: ["admin"],
+                        readers: ["admin"],
+                        timestamp: 1669817887106,
+                        lastEditor: "admin",
+                        canRead: true,
+                        canWrite: true,
+                      },
+                      children: {
+                        "241": {
+                          entry: {
+                            isDirectory: true,
+                            owners: ["admin"],
+                            readers: ["admin"],
+                            timestamp: 1669817950113,
+                            lastEditor: "admin",
+                            canRead: true,
+                            canWrite: true,
+                          },
+                          children: {
+                            "241_metadata": {
+                              entry: {
+                                content: '{"name":"dfgfsdg"}',
+                                isDirectory: false,
+                                owners: ["admin"],
+                                readers: ["admin"],
+                                timestamp: 1669817950113,
+                                lastEditor: "admin",
+                                canRead: true,
+                                canWrite: true,
+                              },
+                            },
+                          },
+                        },
+                        "478": {
```

Comment:
From a quick look it looked like `478` is a child of `346` in `structure`, but of `31b` in `thumbnails`.
Can you double check and adapt if necessary ?

Reply:
Nope, it's a child of `31b` in `structure` too:
![image](https://user-images.githubusercontent.com/15254842/208125066-ece41159-dde2-4e2b-902c-fe58c727a439.png)


![image](https://user-images.githubusercontent.com/15254842/208125163-7e5b1715-dcf6-4318-877d-6d5dd392c27f.png)

Reply:
Thanks for the double check